src/app/groups/containers/user-group-invitations/user-group-invitations.component.html

```diff
@@ -1,8 +1,85 @@
 <h2 class="alg-h2 alg-text-normal alg-base-title-space" i18n>Pending invitations</h2>
-<alg-pending-request
-  [state]="state"
-  [columns]="columns"
-  [requests]="requests"
-  (sort)="onFetch($event)"
-  (processRequests)="onProcessRequests($event)"
-></alg-pending-request>
+@if (state$ | async; as state) {
+  <alg-error
+    *ngIf="state.isError"
+    class="dark"
+    icon="ph-duotone ph-warning-circle"
+    i18n-message message="Error while loading the pending requests."
+  ></alg-error>
+  <p-table
+    class="alg-table"
+    [value]="state.data || []"
+    [loading]="state.isFetching"
+    sortMode="multiple"
+    [customSort]="true"
+    (sortFunction)="onCustomSort($event)"
+  >
+    <ng-template pTemplate="sorticon" let-number>
+      <i class="ph-duotone" [ngClass]="{
+      'ph-arrows-down-up': number === 0,
+      'ph-sort-ascending': number === -1,
+      'ph-sort-descending': number === 1
+    }"></i>
+    </ng-template>
+    <ng-template pTemplate="header">
+      <tr>
+        <th><span i18n>TITLE</span></th>
+        <th><span i18n>TYPE</span></th>
+        <th pSortableColumn="at">
+          <span i18n>REQUESTED ON</span>
```

Comment:
The other column titles seem to be in lowercase

Reply:
Fixed